runconfig/opts/parse.go

```diff
@@ -232,6 +233,7 @@ func AddFlags(flags *pflag.FlagSet) *ContainerOptions {
 	flags.Int64Var(&copts.cpuRealtimePeriod, "cpu-rt-period", 0, "Limit CPU real-time period in microseconds")
 	flags.Int64Var(&copts.cpuRealtimeRuntime, "cpu-rt-runtime", 0, "Limit CPU real-time runtime in microseconds")
 	flags.Int64VarP(&copts.cpuShares, "cpu-shares", "c", 0, "CPU shares (relative weight)")
+	flags.Float64Var(&copts.cpus, "cpus", 0.0, "Number of CPUs")
```

Comment:
This is not a float. It is a fixed point fractional number. This should be the same as `--limit-cpu` in `docker service create/update`.